OpenSim/Common/Storage.cpp

```diff
@@ -1294,7 +1307,6 @@ TimeSeriesTable Storage::exportToTable() const {
     TimeSeriesTable table{};
 
     table.addTableMetaData("header", getName());
-    table.addTableMetaData("version", std::to_string(LatestVersion));
     table.addTableMetaData("inDegrees", std::string{_inDegrees ? "yes" : "no"});
```

Comment:
Not sure why version was removed from meta data, doesn't really matter as long as when we eventually write a file it has the correct version number (so it can be checked if format changes again in the future), can you verify? Thanks

Reply:
This line was causing STO files to contain the following in the header:

```
version=1
...
version=2
```

There is no need to store the version number of the STO file in the TimeSeriesTable, because a TimeSeriesTable is not versioned.

Reply:
Thanks for the clarification @chrisdembia, just making sure that if/when the table is eventually written to a file then it would have the correct version number since the table is not the only client of these files. If that holds then LGTM and feel free to merge.

Reply:
> just making sure that if/when the table is eventually written to a file then it would have the correct version number since the table is not the only client of these files.

Yes, this is the case